addons/web/static/src/xml/notification.xml

```diff
@@ -11,8 +11,8 @@
                 <span class="d-inline" aria-hidden="true">&amp;times;</span>
             </button>
         </t>
-        <div t-if="widget.title" class="toast-header">
-            <span t-attf-class="fa fa-2x mr-3 #{widget.icon} o_notification_icon"
+        <div t-if="widget.title" class="toast-header" t-att-style="widget.titleColor ? _.str.sprintf('color:%s', widget.titleColor) : ''">
```

Comment:
Feels like this is too complex to handle in a template, maybe we should have a `widget.styles` attribute that is computed in JS? This will also allow to more easily add other style properties in the future if we want.